src/websockets-server.hpp

```diff
@@ -63,7 +63,7 @@ class WebsocketServer {
 
   struct lws_protocols m_protocols[3] = {
     {"http-only", &callbackHttp, 32, 0, 0, nullptr, 0},
-    {"od4", &callbackData, sizeof(int32_t), 1024, 0, nullptr, 0},
+    {"od4", &callbackData, sizeof(int32_t), 32183, 0, nullptr, 32183},
```

Comment:
What does the magic number 32,183 represent? Shouldn't we use here UDP_PACKET_MAX (65,535)?

Reply:
Didn't dare. Don't know how if anything extra is reserved by lws... So I just picked half. Perhaps possible to find somewhere.